nncf/quantization/algorithms/weight_compression/activation_stats.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Tuple, TypeVar
+
+from nncf.tensor import functions as fns
+
+TTensor = TypeVar("TTensor")
+
+
+def process_stats(stats: List[TTensor], subset_size: int) -> Tuple[TTensor, TTensor]:
```

Comment:
@andreyanufr, here's the common function for AWQ, Scale Estimation and Lora Correction